source/whatever/index.html

```diff
@@ -9,7 +9,7 @@ <h2><b>Perl&nbsp;6 </b> - Whatever</h2>
 
     <p>
         Everything that can't be categorized otherwise fits into the
-        "Whatever" slot, or as the Perl 6 hacker would write it,
+        "Whatever" slot, or as the Perl 6 hacker would write it,
```

Comment:
Can you perhaps put an actual `&nbsp;` in there?

Reply:
Made this fix: replaces all spaces in "Perl 6".